module/actor/actor-degenesis.js

```diff
@@ -463,12 +463,14 @@ export class DegenesisActor extends Actor {
 
         let cardData = this.constructCardData("systems/degenesis/templates/chat/roll-card.html", DEGENESIS.skills[skill])
 
+        if (type == "initiative")
+            cardData = this.constructCardData("systems/degenesis/templates/chat/initiative-roll-card.html", DEGENESIS.skills[skill] + " - " + "Initiative") 
```

Comment:
I figured there may be ways to use different skills for initiative (I could be wrong), so showing the skill used seems useful.